src/.formatted-files

```diff
@@ -83,7 +83,7 @@ expedited.h
 fastfilter.h
 globals.cpp
 hash.cpp
-hash.h
+hashwrapper.h
```

Comment:
should we also then change the name of hash.cpp to hashwrapper.cpp?

Reply:
I didn't to make the minimum changes possible to make cherrypicking from other projects easier, but I don't have a strong opinion either way so I'll do it.